stackslib/src/net/api/postblock_proposal.rs

```diff
@@ -559,6 +579,8 @@ impl NakamotoBlockProposal {
             });
         }
 
+        let validation_time_ms = time_elapsed();
```

Comment:
This should be rewritten to use `std::time::Instant`:

```rust
let start = Instant::now();
// ...
let validation_time_ms = start.elapsed();
```

`Instant` uses `CLOCK_MONOTONIC` rather than `CLOCK_REALTIME`. The difference is that `CLOCK_REALTIME` tracks wall clock time, so it can jump around or go backwards due to NTP or other adjustments, whereas `CLOCK_MONOTONIC` cannot go backwards. This makes `CLOCK_MONOTONIC` is better for measuring time elapsed

I know I wrote this function initially, so sorry about that!

Reply:
Updated in 6e06712f23a1b0bd695c929065385a5991409fc3.